lib/rules/prefer-true-attribute-shorthand.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @author Pig Fang
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'require shorthand form attribute when `v-bind` value is `true`.',
+      categories: undefined,
+      url: 'https://eslint.vuejs.org/rules/prefer-true-attribute-shorthand.html'
+    },
+    fixable: null,
+    hasSuggestions: true,
+    schema: [],
+    messages: {
+      report:
+        "Boolean prop with 'true' value can be written in shorthand form.",
+      fix: 'Rewrite this prop into shorthand form.'
+    }
+  },
+  /** @param {RuleContext} context */
+  create(context) {
+    // ...
+
+    return utils.defineTemplateBodyVisitor(context, {
```

Comment:
Please remove the boilerplate comment and empty line:

```suggestion
    return utils.defineTemplateBodyVisitor(context, {
```